site/content/docs/main/tilt.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Rapid iterative Velero development with Tilt "
+layout: docs
+---
+
+## Overview
+This document describes how to use [Tilt](https://tilt.dev) with any cluster for a simplified
+workflow that offers easy deployments and rapid iterative builds.
+
+This setup allows for continuing deployment of the Velero server and, if specified, any provider plugin or the restic daemonset.
+It does this work by:
+
+1. Deploying the necessary Kubernetes resources, such as the Velero CRDs and Velero deployment
+1. Building a local binary for Velero and (if specified) provider plugins as a `local_resource`
+1. Invoking `docker_build` to live update any binary into the container/init container and trigger a re-start
+
+Tilt will look for configuration files under `velero/tilt-resources`. Most of the
+files in this directory are gitignored so you may configure your setup according to your needs.
+
+## Prerequisites
+1. [Docker](https://docs.docker.com/install/) v19.03 or newer
+1. A Kubernetes cluster (does not have to be Kind)
```

Comment:
suggest calling out that we support only kubernetes v1.10 and beyond.
Also, the volume provisioners and drivers will not be installed by this setup.

A few notes from my e2e work.

Reply:
Done ✅